src/Sql/Vault/dbo/Stored Procedures/Cipher/Cipher_Delete_V2.sql

```diff
@@ -0,0 +1,73 @@
+﻿CREATE PROCEDURE [dbo].[Cipher_Delete_V2]
+    @Ids AS [dbo].[GuidIdArray] READONLY,
+    @UserId AS UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    CREATE TABLE #Temp
+    (
+        [Id] UNIQUEIDENTIFIER NOT NULL,
+        [UserId] UNIQUEIDENTIFIER NULL,
+        [OrganizationId] UNIQUEIDENTIFIER NULL,
+        [Attachments] BIT NOT NULL
+    )
+
+    INSERT INTO #Temp
```

Comment:
I find joins to have better plans than in clauses this should be the same:

INSERT INTO #Temp
SELECT
    ucd.[Id],
    ucd.[UserId],
    ucd.[OrganizationId],
    CASE WHEN ucd.[Attachments] IS NULL THEN 0 ELSE 1 END
FROM
    [dbo].[UserCipherDetails_V2](@UserId) ucd
INNER JOIN
    @Ids ids ON ucd.[Id] = ids.Id
WHERE
    ucd.[Edit] = 1